apps/angular/src/core/interceptors/animeService.interceptor.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@angular/core';
+import { HttpEvent, HttpInterceptor, HttpHandler, HttpRequest } from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+import { environment } from '@js-camp/angular/environments/environment';
+
+/** Interceptor to handle authorization.  */
+@Injectable()
+export class AnimeServiceInterceptor implements HttpInterceptor {
+
+	/**
+	 * Intercepts the request to the server and adds authorization info.
+	 *  @param request Outgoing request.
+	 *  @param next Sends cloned request with header to the next handler.
+	 */
```

Comment:
```suggestion
        /** @inheritdoc */
```
:)

If I'm not mistaken this method is documented in `HttpInterceptor` interface

Reply:
Alright, thanks for pointing it out!